.github/pull_request_template.md

```diff
@@ -1,6 +1,10 @@
 ## Linear
-<!-- Replace issue id below or add it in the PR title -->
-Fixes LINEAR_ISSUE_ID
+<!--
+  Replace issue id below or add it in the PR title,
+  you can also replace "References" with the following keywords
+  "Completes" or "Fixes" to automatically move the issue to done on merge
```

Comment:
@FrikkieSnyman mentioned `Contributes` this morning. In case of contribution, that feels more natural to me than `References`. 
In practice they do the same on the Linear / GitHub integration, so it doesn't really matter..

Reply:
It would have to be "contributes to" or "contributes towards", so it won't be a single word. I'm totally fine with that, I just chose to use a single word. I'll update the PR, LMK if you prefer a single word and I'll revert it.

> To link PRs or commits to issues without them closing your issue, use one of the contributing magic words next to your issue ID in the PR/Commit description: ref, references, part of, related to, contributes to, towards.

https://linear.app/docs/github#pr-automation > Magic words.

Reply:
Hmm, I hadn't realized that 🤔 now I don't have a preference anymore. I like a single word and I like 'contributes'.

Reply:
Well, my preference is maximizing the work we should not do, and that means I'll keep it like this until someone has a strong preference :D (nice to see you went through the same process I did haha)